### PR TITLE
Add query methods to cosmwasm contract

### PR DESCRIPTION
**Pull request by jayantk**
Add queries for reading the update fee and the valid time period. Use this query in the terra test.

### CODE REVIEW

File: cosmwasm/contracts/pyth/src/msg.rs [hunk 1]
```diff
@@ -14,6 +18,7 @@ use {
 type HumanAddr = String;
 
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
```
Comment (jayantk) [inline cosmwasm/contracts/pyth/src/msg.rs:21]: drive-by: all the other messages have this.

File: cosmwasm/contracts/pyth/src/msg.rs [hunk 2]
```diff
@@ -43,7 +48,17 @@ pub enum ExecuteMsg {
 #[serde(rename_all = "snake_case")]
 pub struct MigrateMsg {}
 
-pub use pyth_sdk_cw::{
-    PriceFeedResponse,
-    QueryMsg,
-};
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
```
Comment (jayantk) [inline cosmwasm/contracts/pyth/src/msg.rs:52]: I've copied all the messages from the sdk into the code here so I can iterate quickly. I'll go back and update the SDK once the contract is done.

File: third_party/pyth/p2w-relay/src/relay/terra.ts [hunk 3]
```diff
@@ -181,18 +181,33 @@ export class TerraRelay implements Relay {
     logger.info("Querying terra for price info for priceId [" + priceId + "]");
 
     const lcdClient = new LCDClient(this.lcdConfig);
-
-    const mk = new MnemonicKey({
-      mnemonic: this.walletPrivateKey,
-    });
```
Comment (jayantk) [inline third_party/pyth/p2w-relay/src/relay/terra.ts:187]: unused variable

File: cosmwasm/contracts/pyth/src/contract.rs [hunk 4]
```diff
@@ -846,6 +868,65 @@ mod test {
...
+                fee_denom: fee_denom.clone(),
+                ..create_zero_config_info()
+            })
+            .unwrap();
+
+        assert_eq!(
+            get_update_fee(deps.as_ref(), &updates[0..1]),
+            Ok(Coin::new(big_fee.u128(), fee_denom))
+        );
+        assert!(get_update_fee(deps.as_ref(), &updates[0..2]).is_err());
```
Comment (ali-bahjati) [inline cosmwasm/contracts/pyth/src/contract.rs:911]: Nitpick: Can't we specify the exact error here?

File: cosmwasm/contracts/pyth/src/msg.rs (hunk 2 above)
Comment (ali-bahjati) [reply cosmwasm/contracts/pyth/src/msg.rs:52]: Or remove the sdk entirely :D

**Review by ali-bahjati — APPROVED** [review]
Nice!
